prediction/src/algorithms/evaluation/evaluation.py

```diff
@@ -1,19 +1,26 @@
 """
 This file contains logic to evaluate the prediction models.
 """
+import glob
 import os
+from tqdm import tqdm
+import sys
+from collections import defaultdict
 
 import numpy as np
 import pylidc as pl
-from tqdm import tqdm
-from src.algorithms.classify import trained_model
-from src.algorithms.evaluation.metrics import get_accuracy, logloss
-from src.preprocess.lung_segmentation import get_z_range
 
+sys.path.insert(0, os.path.abspath(os.path.join(__file__, '..', '..', '..', '..')))
```

Comment:
why is this necessary?

Reply:
I kept getting `SystemError: Parent module '' not loaded, cannot perform relative import` so I sticked to [this SO answer](https://stackoverflow.com/questions/16981921/relative-imports-in-python-3). It adds the absolute path of the `prediction` folder to the path which makes it possible to import the desired methods properly.

Reply:
Hmmm, i did noticed that your other PR had a `__main__` section, which fits with that SO question about running scripts. I didn't see a similar script runner in this one though. Personally, I've had issues when switching bw running modules inside vs outside of docker. That's what led me to add the `try/except` code around the config imports.

Reply:
I removed the `__main__` section because developers are also able to choose which evaluation method to run by e.g.
`python -c "from prediction.src.algorithms.evaluation.evaluation import evaluate_segmentation;evaluate_segmentation()"`.
By adapting the test you provided in #290 one can also import and run the evaluation method using docker :)

Reply:
This does look really wrong. :p  (I mean, at the very least it should use `dirname`!)

Reply:
@lamby Adapted. Thank you!